outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "role_arn" {
```

Comment:
a thought: Since we're using slightly different names for these in the docs ("Integration Role ARN" and "External ID") it might be nice to add `description` attributes for these.